product_pricelist_item_sequence/models/product_pricelist_item.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019-2025 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+    # NOTE: we do not change _order, we are just adding sequence
+    # and visualizing a different order in view.
+
+    ppi_sequence = fields.Integer(default=16)
```

Comment:
maybe just `sequence` is better? Was thinking about it.. on one hand it avoids conflicts to have it unique, on the other hand, this field name is so standard that it's probably what everyone would choose and then it plays well with other modules implement the same